etc/poco/bundle/main.json

```diff
@@ -51,7 +51,8 @@
         "/api/**"
       ],
       "methods": [
-        "*"
+        "GET",
```

Comment:
I had a look at all routers we have that do a DELETE call and the only endpoints are:
/deleteInstallation 
/:jiraHost
/repo-sync-states

Also had a look in Splunk and those other endpoints haven't been called in a month so I think it's ok to restrict for now and if we do come across the need to allow one or both of them again, we can do so.